src/main/scala/is/hail/sparkextras/ContextRDD.scala

```diff
@@ -94,25 +94,26 @@ class ContextRDD[C <: AutoCloseable, T: ClassTag](
   def collect(): Array[T] =
     run.collect()
 
+  // WARNING: this resets the context, when this method is called, the value of
+  // type `T` must already be "stable" i.e. not dependent on the region
+  private[this] def decontextualize(c: C, it: Iterator[C => Iterator[T]]): Iterator[T] =
+    new SetupIterator(it.flatMap(_(c)), c.reset _)
```

Comment:
no need to create anonymous function, delete ` _` after `reset`. (same with line 225)